DfE.FindInformationAcademiesTrusts.Data.AcademiesDb/Factories/AcademyFactory.cs

```diff
@@ -0,0 +1,34 @@
+using System.Globalization;
+using DfE.FindInformationAcademiesTrusts.Data.AcademiesDb.Extensions;
+using DfE.FindInformationAcademiesTrusts.Data.AcademiesDb.Models;
+
+namespace DfE.FindInformationAcademiesTrusts.Data.AcademiesDb.Factories;
+
+public interface IAcademyFactory
+{
+    Academy CreateAcademyFrom(GroupLink gl, Establishment establishment);
+}
+
+public class AcademyFactory : IAcademyFactory
+{
+    public Academy CreateAcademyFrom(GroupLink gl, Establishment establishment)
+    {
+        return new Academy(
+            establishment.Urn,
+            DateTime.ParseExact(gl.JoinedDate!, "dd/MM/yyyy", CultureInfo.InvariantCulture),
```

Comment:
can you use your new date extension here?

Reply:
This one is a non-nullable date so doesn't need the extension method :-)